addons/storyshots/storyshots-core/src/frameworks/angular/renderTree.ts

```diff
@@ -7,35 +7,35 @@ import { BrowserDynamicTestingModule } from '@angular/platform-browser-dynamic/t
 // eslint-disable-next-line import/no-extraneous-dependencies
 import { NO_ERRORS_SCHEMA } from '@angular/core';
 import { addSerializer } from 'jest-specific-snapshot';
-import { initModuleData } from './helpers';
+import { RenderNgAppService } from '@storybook/angular/dist/client/preview/angular-beta/RenderNgAppService';
```

Comment:
Avoids redefining the rendering like before.
if you have a (simple) idea to avoid the `/dist/` tell me :)